pkg/deploy/controller/deployment/factory.go

```diff
@@ -51,6 +51,7 @@ func (factory *DeploymentControllerFactory) Create() controller.RunnableControll
 
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartRecordingToSink(factory.KubeClient.Events(""))
+	eventRecorder := eventBroadcaster.NewRecorder(kapi.EventSource{Component: "deployments-controller"})
```

Comment:
@smarterclayton any better name for our controller? "deployment-controller" conflicts with upstream.